### ISSUE TITLE
responsive_mockups should wait for the page to be actually loaded before taking a screenshot

### ISSUE THREAD

**Issue by ChrisKam** [fabrik42/responsive_mockups#4]
In order to generate any kind of output I had to increase the setTimeout in the index.js to around 5 seconds. While my websites most certainly don't need 5 seconds to fully load, I believe this has something to do with typekits preferred way of prevents FOUCs - by having the page fade in only after all webfonts have loaded.  Maybe some config option for such cases might help here; it doesn't seem to be struggling with images, even larger ones.

Thanks gain for your work with this script, Christian! I'm really looking forward to putting it to good use.

**Comment by fabrik42** [comment]
Does this invalidate this issue? https://github.com/fabrik42/responsive_mockups/issues/3

**Comment by fabrik42** [comment]
The screenshots are done 1 second after the `page.onLoadFinished` fires - so the website should be loaded by then. There maybe a problem if your layout relies on styles/fonts that are loaded in a deferred way.

I agree, an additional option to increase the waiting time would make sense.

**Comment by ChrisKam** [comment]
The page I'm testing this out on is http://relaunch.ksb-wesel.de - if you check the networks tabs, you can see that typekit finishes loading well before 1s after page load, but still nothing would show up for me at the standard 1s. I'm running this on Windows 7 64 bit btw

edit: I had overlooked something - this _might_ be Typekit related after all. It seems like the conditions for Typekit displays aren't met in headless browsers like you were saying, and thus the page tries loading longer and only fades in as a sort of fail-safe if typekit is down.

**Comment by fabrik42** [comment]
Yes, this may be the case.

**Comment by fabrik42** [comment]
Closing this. Please reopen, if there is anything new to do.
